ui/app/AppLayouts/Profile/views/EnsDetailsView.qml

```diff
@@ -120,7 +120,8 @@ Item {
             sendType: Constants.SendType.ENSRelease
             preSelectedRecipient: root.ensUsernamesStore.getEnsRegisteredAddress()
             preDefinedAmountToSend: LocaleUtils.numberToLocaleString(0)
-            preSelectedAsset: store.getAsset(releaseEnsModal.store.assets, "ETH")
+            preSelectedHolding: store.getAsset(releaseEnsModal.store.assets, "ETH")
+            preSelectedHoldingType: HoldingTypes.Type.Asset
```

Comment:
We could use the existing sendType by adding a new value as I have done in PR already instead of creating this new preSelectedHoldingType

Reply:
As discussed, we'd rather keep the new enum exclusively for the Holding type, as opposed to the transaction type